server/src/data/measurables-data.js

```diff
@@ -0,0 +1,156 @@
+const Sequelize = require('sequelize');
+
+const models = require("../models");
+const { notify } = require("../lib/notifications");
+
+const { UsersData } = require('./users-data');
+
+class MeasurablesData {
+
+  constructor() {
+    this.usersData = new UsersData();
+  }
+
+  /**
+   * @param root
+   * @param values
+   * @param options
+   * @return {Promise<*>}
+   */
+  async createMeasurable (root, values, options) {
+    const {
+      name,
+      description,
+      valueType,
+      expectedResolutionDate,
+      resolutionEndpoint,
+      descriptionDate,
+      descriptionEntity,
+      descriptionProperty,
+      channel
+    } = values;
+    let _auth0Id = await this.usersData.getAuth0Id(options);
+    const user = await this.usersData.auth0User(_auth0Id);
```

Comment:
While this is being done, can this be DRYd up?

Reply:
let's refactor and write tests for that in a separate task

Reply:
Sounds good